pc/device.c

```diff
@@ -310,16 +319,18 @@ int ctap_user_verification(uint8_t arg)
 
 uint32_t ctap_atomic_count(int sel)
 {
-    static uint32_t counter1 = 25;
-    /*return 713;*/
+    // Replaced with global counter1 -- read from file
+    // static uint32_t counter1 = 25; -- this will cause failures when executable is re-invoked
     if (sel == 0)
     {
         printf1(TAG_RED,"counter1: %d\n", counter1);
-        return counter1++;
+	counter1++; // increment global counter
```

Comment:
The code is already clear and doesn't need comment.  Also the white space is inconsistent, please adjust it.